thetagang.toml

```diff
@@ -467,7 +467,9 @@ logfile = '/etc/thetagang/ib_async.log'
 # Typically the amount of time needed when waiting on data from the IBKR API.
 # Sometimes it can take a while to retrieve data, and it's lazy-loaded by the
 # API, so getting this number right is largely a matter of guesswork.
-api_response_wait_time = 60
+# Increasing the number would reduce the throughput since we have to wait X seconds
+# to make sure all the required data fields are ready.
+api_response_wait_time = 10
```

Comment:
I'm trying to understand this change, and it looks to me like the waiting behaviour is the same as before (i.e., it will stop waiting once the condition is met, or the timer expires, whichever happens first).

The problem is that IBKR occasionally takes a really long time to return data, which causes wonky behaviour, so I used a fairly big timeout here for safety.

I would prefer to keep a longer timeout if it doesn't have a negative impact (it should return quickly under normal circumstances when the data arrives quickly).

I'm assuming we still won't have to wait the entire 10 or 60 seconds for each request, correct?

Reply:
that's a good question, I think it really depends on the field that query.

it will wait up to the `api_response_wait_time` to make sure both required/optional fields are ready, so it is still possible that data is missing after `api_response_wait_time`.

I will change this back to 60 seconds and update the comment to indicate if users want higher throughput, they can reduce this number, and the tradeoff would be less accurate data.